src/pages/edge/adobe-journey-optimizer/in-app-message/tutorials/handle-clicks.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
following 5 files
- handle-clicks.md
- javascript-from-native.md
- messaging-delegate.md.  + (tab file)
- native-from-javascript.md. + (tab file)
- javascript-from-native.md
- validate-message.md 

are not newly created. They are existing files. They are unchanged, moved from the tutorials folder to in-app-message/tutorial folder.